package.json

```diff
@@ -6,7 +6,9 @@
   "scripts": {
     "build": "gatsby build",
     "dev": "gatsby develop",
-    "clean": "gatsby clean"
+    "clean": "gatsby clean",
+    "format": "prettier \"src/**/*.{ts,tsx,html}\" --write",
+    "lint": "eslint \"src/**/*.{ts,tsx}\" --quiet"
```

Comment:
(Optional)
Ever heard of precommit? Rather than using manually format, it can be automatically format before commit

Reply:
Ups, use format on precommit. lint for test

Reply:
Does it refer to GitHooks? Never used it, but I think it's a good idea to remove manual works

Reply:
Use wrapped version here: https://github.com/okonet/lint-staged it maybe not too flexible but it's easy to configure it, 
```
{
  "scripts": {
    ..,
    "precommit" : "lint-staged"
    },
   "lint-staged": {
    "*.{ts,tsx}": [
      "yarn prettier",
      "yarn lint",
      "git add"
    ]
  }
}
```

Reply:
I used pretty-quick and husky alongside lint-staged for pre-commit formatting, check it on here https://github.com/deeptechid/deeptech.id/pull/97/commits/3969b465b913d40526045fd4273fffa93963a3b1